validation/Validation.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+#include <Global.test.hpp>
+
+class ValidationTestDefinitions : public ::testing::Test
+{
+   protected:
+    struct TestCase
+    {
+        std::string testName_;
+        std::vector<std::string> compareWith_;
+        std::vector<double> tolerance_;
+    };
+
+    void SetUp() override
+    {
+        // List of validation test scenarios go here (replacement for a "super yaml file", althought this could be added
+        // in the future)
+        const std::vector<TestCase> scenarios_ = {
+            {"SCENARIO_1: Spherical gravity only", {"GMAT", "OREKIT"}, {1.0e-6, 1.0e-6}},
```

Comment:
Should the friendly scenario name be part of the yaml?

Also, how does this link to the yaml/csv file name?

Reply:
I agree with having a human-friendly scenario name 👍 

What about this? 

> Also, how does this link to the yaml/csv file name?

Where do you specify the actual file name, e.g. `scenario-001.` ?

Maybe I'm getting ahead/missing something.

Reply:
I'll just parse it out of the string or something like that to indicate the filename